pkgs/development/haskell-modules/configuration-nix.nix

```diff
@@ -720,6 +720,17 @@ self: super: builtins.intersectAttrs super {
     '';
   }) super.sbv;
 
+  rest-rewrite = overrideCabal (drv: {
+    testToolDepends = (drv.testToolDepends or [ ]) ++ [ pkgs.graphviz pkgs.z3 ];
+
+    # Test suite 'rest' relies on the relative dir graphs/ existing. This
+    # is a problem for "out of tree" builds, as the directory is not copied
+    # over.
+    postPatch = ''
+      mkdir graphs
+    '';
```

Comment:
Could you report this upstream, and add a link to the issue here in this comment?

If this directory is necessary for the tests, then it either needs to be in the .cabal file, or explicitly created by the test suite.  Otherwise, building from the sdist won't work (like in the case of trying to do a build from `stack unpack`).